Don't force sdk 2 (through gmcs), e.g. gtk-sharp 2.12 uses sdk 4 cairo.
--- mono-tools-2.10/mperfmon/Makefile.am.orig	2011-02-12 17:32:47.000000000 +0100
+++ mono-tools-2.10/mperfmon/Makefile.am	2014-01-11 11:28:30.148463700 +0100
@@ -29,5 +29,5 @@
 ress= $(foreach res,$(mperfmon_resources), $(addprefix -resource:,$(res)),$(notdir $(res)))
 
 mperfmon.exe: $(mperfmon_sources) $(mperfmon_resources)
-	$(GMCS) -noconfig -codepage:utf8 -warn:4 -optimize+ -debug -define:DEBUG -target:exe -out:mperfmon.exe $(mperfmon_sources) $(ress) $(packages) $(assemblies)
+	$(MCS) -noconfig -codepage:utf8 -warn:4 -optimize+ -debug -define:DEBUG -target:exe -out:mperfmon.exe $(mperfmon_sources) $(ress) $(packages) $(assemblies)
 
--- mono-tools-2.10/Mono.Profiler/Mono.Profiler.Widgets/Makefile.am.orig	2014-01-11 14:36:07.162081890 +0100
+++ mono-tools-2.10/Mono.Profiler/Mono.Profiler.Widgets/Makefile.am	2014-01-11 14:50:26.770500642 +0100
@@ -56,7 +56,7 @@
 
 $(ASSEMBLY) $(ASSEMBLY_MDB): $(build_sources) $(build_resources)
 	mkdir -p $(BUILDDIR)
-	$(GMCS) $(CSFLAGS) -out:$(ASSEMBLY) -target:library $(build_sources) $(build_resources_args) $(build_references)
+	$(MCS) $(CSFLAGS) -out:$(ASSEMBLY) -target:library $(build_sources) $(build_resources_args) $(build_references)
 
 EXTRA_DIST = Mono.Profiler.Widgets.csproj $(FILES) $(RESOURCES)
 
